lib/Tool/Requirements.php

```diff
@@ -609,6 +609,13 @@ public static function checkPhp()
             ]);
         }
 
+        $checks[] = new Check([
+            'name' => 'locales-utf8',
+            'link' => 'https://packages.debian.org/en/stable/locales-all',
+            'state' => setlocale(LC_ALL, ['en.utf8', 'en_US.utf8', 'en_GB.utf8']) === false ? Check::STATE_ERROR : Check::STATE_OK,
+            'message' => "UTF-8 locale is not available, thus all CLI calls which use escapeshellarg() will strip multibyte characters",
```

Comment:
```suggestion
            'message' => "It is recommended to install UTF-8 locale, otherwise all CLI calls which use escapeshellarg() will strip multibyte characters",
```
right now, the message is a bit strange... 
![image](https://github.com/pimcore/pimcore/assets/8792145/6d291d15-1469-482a-9f3f-94b2c78e8bb1)

Reply:
Thanks, @fashxp. Message changed as you proposed.